app/controllers/volunteers_controller.rb

```diff
@@ -12,7 +12,7 @@ def create
     @volunteer = Volunteer.new(create_volunteer_params)
 
     if @volunteer.save
-      VolunteerMailer.account_setup(@volunteer).deliver
```

Comment:
Is there any reason to keep `VolunteerMailer#account_setup` around or can we replace all instances of it with `DeviseInvitable`?

Reply:
We left `VolunteerMailer#account_setup` because it's doing more than just sending an email when a volunteer is activated...the `account_setup` method is also calling `generate_password_reset_token` on the user. Sorry, I didn't mention this in the PR...I left a comment about this in the issue here: https://github.com/rubyforgood/casa/issues/751#issuecomment-697502598

Reply:
Ah! That makes perfect sense, I should've checked the issue as well, thanks for the clarification 👌🏼